_partials/_tiered-storage-billing.md

```diff
@@ -0,0 +1 @@
+Because of the proprietary format used to store data in Amazon S3, $COMPANY charges for tiered data based on its uncompressed size, even if you compress it first.
```

Comment:
```suggestion
The data is stored in [Apache Parquet](https://parquet.apache.org/) in Amazon S3, and $COMPANY charges only for the storage it occupies. The original size in Postgres does not directly correspond to its size in S3, as it depends on data locality to achieve optimal compression with Parquet. This applies to both compressed and uncompressed tables. Moreover, a compressed Hypertable in TimescaleDB may require less disk space than its Parquet format in S3. Apart from S3 storage costs, no additional expenses, such as data transfer or compute costs, are passed on to the customer.
```

Reply:
@philkra Oh it's great to have those details, thank you! I have previously structured it in a way that the same sentence would appear in 3 places, which is not ideal with a longer body of text. So I changed this to:

1. Explain the format and the billing in more detail, using your words, on the about page
2. Only add a one-sentence explanation to the pricing and manage tiering pages

Hope this works.